crates/cli/src/bin/wasm-bindgen-test-runner/headless.rs

```diff
@@ -121,7 +121,8 @@ pub fn run(server: &SocketAddr, shell: &Shell, timeout: u64) -> Result<(), Error
 
     // Visit our local server to open up the page that runs tests, and then get
     // some handles to objects on the page which we'll be scraping output from.
-    let url = format!("http://{}", server);
+    let url =
+        std::env::var("WASM_BINDGEN_TEST_ADDRESS").unwrap_or_else(|_| format!("http://{}", server));
```

Comment:
```suggestion
    let url = format!("http://{}", std::env::var("WASM_BINDGEN_TEST_ADDRESS").unwrap_or(server));
```
I was briefly thinking that this might be a good idea, but maybe there is a use case out there for `https`?